starboard/BUILD.gn

```diff
@@ -170,17 +177,36 @@ source_set("starboard_headers_only") {
   }
 }
 
-if (platform_tests_path == "") {
-  # If 'starboard_platform_tests' is not defined by the platform, then an
-  # empty 'starboard_platform_tests' target is defined.
-  target(gtest_target_type, "starboard_platform_tests") {
-    testonly = true
-
-    sources = [ "//starboard/common/test_main.cc" ]
-
+if (current_toolchain == starboard_toolchain) {
```

Comment:
It can be if you feel strongly about it, but the main purpose of that file was to add `is_starboardized_toolchain`--the rest of the variables are pretty static and I don't think it reads that different to see this conditional vs `if (is_starboard_toolchain)`